src/Plugins/Windows/nowide/cstdio.hpp

```diff
@@ -144,22 +144,24 @@ inline DIR* opendir(char const *name)
     return _wopendir (wname.c_str());
 }
 
-inline char* readir_entry(DIR* dir)
```

Comment:
Plugins/Windows/nowide is imported from Boost Nowide

It is not a good practice to modify the Nowide library directory. We could write a TeXmacs internal readir_entry somewhere else.